### PR TITLE
Noble tweaks

### PR DESCRIPTION
**Pull request by MemedHams**
## About The Pull Request
A re-upload of my More Noble Options PR, with the offending IRL weapon relics stripped away.
-Adds a few additional items to the default noble weapon pool.
-Neotheologists now have a 50% chance to spawn with a Neotheo-themed heirloom.
-Noble items are now labeled as "[item name] of [family]", making them identifiable on sight.
-Noble items now have a randomly higher price tag, encouraging petty theft and pawning of the priceless relics. Keep your treasure safe!
## Why It's Good For The Game
More funny stuff to steal from noble players, and more impetus for them to live in fear

## Changelog
:cl:
add: nobles who spawn as part of Neotheology now get a 50% chance of aquiring a Neotheo heirloom
tweak: more items for the basic heirloom pool
tweak: noble heirloom names are now visibly structured "(item) of (family)"
tweak: noble heirlooms are now considerably more expensive. Watch your pockets!
/:cl:

### CODE REVIEW

File: code/datums/perks/fate.dm [hunk 1]
```diff
@@ -132,15 +132,32 @@
 		holder.stats.removePerk(src.type)
 		return
 	var/turf/T = get_turf(holder)
-	var/obj/item/W = pickweight(list(
+	var/obj/item/W = null
```
Comment (TheShown911) [inline code/datums/perks/fate.dm:135]:
```suggestion
	var/obj/item/W
```

File: code/datums/perks/fate.dm [hunk 2]
```diff
@@ -132,15 +132,32 @@
...
-				/obj/item/tool/knife/dagger/ceremonial = 0.8,
+				/obj/item/tool/knife/dagger = 0.5,
+				/obj/item/gun/projectile/colt = 0.2,
+				/obj/item/gun/projectile/revolver/havelock = 0.1,
+				/obj/item/tool/knife/dagger/ceremonial = 0.4,
 				/obj/item/gun/projectile/revolver = 0.4))
 	holder.sanity.valid_inspirations += W
 	W = new W(T)
 	W.desc += " It has been inscribed with the \"[holder.last_name]\" family name."
+	W.name = "[W] of [holder.last_name]"
```
Comment (TheShown911) [inline code/datums/perks/fate.dm:160]: Does this interact well with weapon mods? I remember them using initial() but that might have been fixed.

File: code/datums/perks/fate.dm [hunk 3]
```diff
@@ -152,6 +169,7 @@
 	W.AddComponent(/datum/component/inspiration, final_oddity, get_oddity_perk())
 	W.AddComponent(/datum/component/atom_sanity, 1, "") //sanity gain by area
 	W.sanity_damage -= 1 //damage by view
+	W.price_tag += rand(2500,5000)
```
Comment (TheShown911) [inline code/datums/perks/fate.dm:172]: Loadsamoney

**Review by TheShown911 — APPROVED** [review]
Don't like decimal pickweights, but that's how it was before.

File: code/datums/perks/fate.dm (hunk 3 above)
Comment (hyperioo) [reply code/datums/perks/fate.dm:172]: Yeah that's a lot of money, maybe (500, 1500) is enough

File: code/datums/perks/fate.dm (hunk 2 above)
Comment (MemedHams) [reply code/datums/perks/fate.dm:160]: gun mods seem to be fine
tool mods however seem to cleanly override the new naming, which is sad but not the worst way that could go down.
Is there any decent fix to that?

Comment (MemedHams) [reply code/datums/perks/fate.dm:160]: @hyperioo meant to ping you on above, sorry

File: code/datums/perks/fate.dm (hunk 3 above)
Comment (MemedHams) [reply code/datums/perks/fate.dm:172]: hmm, how about 1000, 2500?

Comment (MemedHams) [reply code/datums/perks/fate.dm]: reduced to above.

File: code/datums/perks/fate.dm [hunk 4]
```diff
@@ -152,6 +169,7 @@
 	W.AddComponent(/datum/component/inspiration, final_oddity, get_oddity_perk())
 	W.AddComponent(/datum/component/atom_sanity, 1, "") //sanity gain by area
 	W.sanity_damage -= 1 //damage by view
+	W.price_tag += rand(1000,2500)
```
Comment (SirRichardFrancis) [inline code/datums/perks/fate.dm:172]:
```suggestion
	W.price_tag += rand(1000, 2500)
```

**Review by SirRichardFrancis — APPROVED** [review]

File: code/datums/perks/fate.dm (hunk 2 above)
Comment (hyperioo) [reply code/datums/perks/fate.dm:160]: @MemedHams You could check how it's done for the gun to check why it's working properly with them and not with the tool. Toolmods are likely based on initial() value of `name` so they don't remember your custom name.

**Pull request merged**